features/upgrade/routing/upgrade.feature

```diff
@@ -208,3 +208,82 @@ Feature: Routing and DNS related scenarios
     Then the step should succeed
     And the output should contain "Hello-OpenShift web-server-rc"
     """
+
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  Scenario: Unidling a route work without user intervention - prepare
+    Given I switch to first user
+    And I run the :new_project client command with:
+      | project_name | ocp45955 |
+    Then the step should succeed
+    When I use the "ocp45955" project
+    Given I obtain test data file "routing/web-server-rc.yaml"
+    When I run the :create client command with:
+      | f | web-server-rc.yaml|
+    Then the step should succeed
+
+    And a pod becomes ready with labels:
+      | name=web-server-rc |
+    Then the expression should be true> service('service-unsecure').exists?
+    When I expose the "service-unsecure" service
+    Then the step should succeed
+
+    Given I wait until number of replicas match "1" for replicationController "web-server-rc"
+    Given I have a test-client-pod in the project
+    When I execute on the pod:
+      | curl |
+      | -ksS |
+      | http://<%= route("service-unsecure", service("service-unsecure")).dns(by: user) %>/ |
```

Comment:
`(by: user)` is no longer needed.
BTW, I think we read in horizontally. So perhaps `| curl | -ksS | url |` ?

Reply:
done

Reply:
Didn't see the changes. Maybe you forgot to push the new changes ?

Reply:
done